pkg/virt-handler/device-manager/device_controller.go

```diff
@@ -134,6 +134,7 @@ type DeviceController struct {
 	stop                chan struct{}
 	mdevTypesManager    *MDEVTypesManager
 	clientset           k8scli.CoreV1Interface
+	mdevRefreshWG       *sync.WaitGroup
```

Comment:
```
Previous write at 0x00000458e690 by goroutine 189:
  kubevirt.io/kubevirt/pkg/virt-handler/device-manager.init.func1.3()
      pkg/virt-handler/device-manager/device_controller_test.go:125 +0xfd
  kubevirt.io/kubevirt/vendor/github.com/onsi/ginkgo/v2/internal.extractBodyFunction.func3()
      vendor/github.com/onsi/ginkgo/v2/internal/node.go:475 +0x2e
  kubevirt.io/kubevirt/vendor/github.com/onsi/ginkgo/v2/internal.(*Suite).runNode.func3()
      vendor/github.com/onsi/ginkgo/v2/internal/suite.go:894 +0x12b
```
Doesn't this mean this is just unit test issue where we reassign the handler sooner than the previous test finishes?

Reply:
Do we think such case is possible?  As I see it the problem is that DeviceHandler is global used across multiple functions. Do we need that?